editor/src/components/canvas/controls/select-mode/absolute-resize-control.tsx

```diff
@@ -200,6 +225,7 @@ const ResizePoint = React.memo(
           }}
         />
         <div
+          onClick={onEdgeDblClick}
```

Comment:
I think we should always use `onDoubleClick` when listening to double click events.

listening to click plus event.detail===2 may trigger under this _wrong_ circumstance:
1. click anywhere in the browser, on the canvas etc. for exmaple: click on the element to select it!
2. then click on the resize control within the doubleclick timeout.  (happens surprisingly more frequently than you might assume. especially if someone is in the flow, and knows what they are doing)

Detail will be 2, you will dispatch the dbclick behavior.

Reply:
great idea! implemented in https://github.com/concrete-utopia/utopia/pull/3269/commits/e4731a5865e9fa45de452eac7a7609ca7dc8aad6